src/app.js

```diff
@@ -2,20 +2,8 @@
 
 const _ = require('lodash')
 const co = require('co')
-const metadata = require('./controllers/metadata')
```

Comment:
Is there new way to get metadata/health/pairs with the connector messaging? It seems like you should still be able to query health and trading pairs